server/src/main/java/org/candlepin/policy/js/entitlement/EntitlementRules.java

```diff
@@ -959,6 +970,9 @@ private void adjustHostedBonusPoolQuantity(PoolManager poolManager, Consumer c,
     public void postEntitlement(PoolManager poolManager, Consumer consumer,
         Map<String, Entitlement> entitlements, List<Pool> subPoolsForStackIds, boolean isUpdate,
         Map<String, PoolQuantity> poolQuantityMap) {
+
+        // TODO: Why are we calling a separate method here instead of just doing that method's work here?
```

Comment:
Nitpick: only formatting changes and this comment in this file - should happen in a separate PR...